assemblies/DataSources/OpenConfig/Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.xml

```diff
@@ -0,0 +1,843 @@
+            <summary>
+            Gets the ID of the table parameter.
+            </summary>
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.PathYangModelAsString">
+            <summary>
+            Gets the path in the YANG model in string format.
+            </summary>
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.PathYangModel">
+            <summary>
+            Gets the path in the YANG model.
+            </summary>
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.Rows">
+            <summary>
+            Gets the mapping between the primary key and the row content.
+            </summary>
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.DeletedRows">
+            <summary>
+            Gets the mapping between the primary key and when the row is deleted.
+            </summary>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.ChangeReferencedValues(System.DateTime,Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Enums.DataValueOriginType)">
+            <summary>
+            Changes the values of referenced columns when one of its source columns has had a value change.
+            </summary>
+            <param name="timestampUtc">UTC time when the value was taken.</param>
+            <param name="origin">Indication of where the data originated from.</param>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.ChangeValue(Gnmi.PathElem[],Gnmi.TypedValue,System.DateTime,Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Enums.DataValueOriginType,System.String)">
+            <summary>
+            Processes the update and adds it to the table cells that need to be changed.
+            </summary>
+            <param name="pathElems">A collection of all path elements.</param>
+            <param name="updateValue">Value that needs to be changed.</param>
+            <param name="timestampUtc">UTC time when the value was taken.</param>
+            <param name="origin">Indication of where the data originated from.</param>
+            <param name="subscriptionName">Name of the subscription in case the data came in through subscription.</param>
+            <returns>Boolean indicating if there is something that needs to be changed.</returns>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.DeleteRow(System.String,System.DateTime)">
+            <summary>
+            Deletes the row when the timestamp is most recent.
```

Comment:
This is a slightly confusing sentence. Do you mean this?

```suggestion
            Deletes the row with the most recent timestamp.
```

Reply:
```suggestion
            Deletes the row if <paramref name="timestampUtc"/> is more recent than the last received one.
```
It will delete the row if the timestamp that is passed along is more recent than the one that is stored in memory. If not it will actually not do anything.